wom/src/main/scala/wom/package.scala

```diff
@@ -105,3 +105,7 @@ final case class CommandSetupSideEffectFile(file: WomFile, relativeLocalPath: Op
 // by the AST. However, it is tricky to get full information from Hermes
 // at the moment.
 final case class SourceFileLocation(startLine: Int)
+
+object SourceFileLocation {
+    def convert(i : Int) : SourceFileLocation = SourceFileLocation(i)
```

Comment:
I'm not sure how writing `val x = SourceFileLocation.convert(i)` is easier than just `val x = SourceFileLocation(i)`?